src/fastcs/util.py

```diff
@@ -3,3 +3,42 @@ def snake_to_pascal(input: str) -> str:
     return "".join(
         part.title() if part.islower() else part for part in input.split("_")
     )
+
+def unpack_status_arrays(parameter: list, uri: list[list[str]]):
+    """Takes a list of parameters and a list of special uri. Search the parameter
+    for elements that match the values in the uri list and split them into one
+    new odinParameter for each value.
+
+    Args:
+        parameter: List of parameters
+        uri: List of special uris to search and replace
+
+    Returns:
+        list[parameters]
+
+    """
+    removelist = []
+    for el in parameter:
+        if el.uri in uri:
+            status_list = (
+                el.metadata["value"]
+                .replace(",", "")
+                .replace("'", "")
+                .replace("[", "")
+                .replace("]", "")
+                .split()
+            )
```

Comment:
This is a bit horrible and I don't think it is necessary because it will populate the value on the initial poll. The key thing is to make the separate parameters with the index on the end of the URI. Could you try setting the value to `""` and then checking that it is updated once the poll loop starts?

Then you also construct the metadata dict once and use it for all the new parameters.